### PR TITLE
Add zh_tw translation

### PR DESCRIPTION
**Pull request by IamNotBrianZ**
## What
zh_tw translation for someone who uses Traditional Chinese

## Implementation Details
Translated from zh_cn by a script
Replace some words for small differences between zh_tw and zh_cn

## Outcome
add zh_tw.json file

## Additional Information
nope

## Potential Compatibility Issues
nope

**Please fill in as much useful information as possible. Also, please remove all unused sections, including this and the other explanations.**

### CODE REVIEW

**Review by screret — APPROVED** [review]

File: src/main/resources/assets/gtceu/lang/zh_tw.json [hunk 1]
```diff
@@ -0,0 +1,5386 @@
...
+    "material.gtceu.creosote":"雜酚油",
+    "material.gtceu.cumene":"異丙苯",
+    "material.gtceu.cupric_oxide":"氧化銅",
+    "material.gtceu.cupronickel":"白銅",
+    "material.gtceu.curium":"鋦",
+    "material.gtceu.cyan_dye":"青色染料",
+    "material.gtceu.cyclohexane":"環己烷",
+    "material.gtceu.cyclohexanone_oxime":"環己酮肟",
+    "material.gtceu.damascus_steel":"大馬士革鋼",
+    "
```
Comment (Chococar-NSW) [inline src/main/resources/assets/gtceu/lang/zh_tw.json]: 好奇問一下，為什麼"錀"是被拆分
因為測試中無法正常顯示嗎?

Comment (Chococar-NSW) [inline src/main/resources/assets/gtceu/lang/zh_tw.json]: 還有"鑪"

Comment (Chococar-NSW) [inline src/main/resources/assets/gtceu/lang/zh_tw.json:5046]: "𨭎"反而沒甚麼問題，因為新注音也無法正常打出來

Comment (Chococar-NSW) [inline src/main/resources/assets/gtceu/lang/zh_tw.json:5277]: RPM真要翻譯建議以"每分鐘轉速"

**Comment by Chococar-NSW** [comment]
Weird, my comment is in right position in review page.
But everything wrong in the pr conversation.

File: src/main/resources/assets/gtceu/lang/zh_tw.json (hunk 1 above)
Comment (IamNotBrianZ) [reply src/main/resources/assets/gtceu/lang/zh_tw.json]: 因為簡體中文的文件是這樣翻的，我再處理一下
如果還有其他的也請和我說

Comment (IamNotBrianZ) [reply src/main/resources/assets/gtceu/lang/zh_tw.json]: 我是用script自動翻的，所以還會有些問題

**Review by Chococar-NSW — APPROVED** [review]
As a Traditional Chinese native speaker, LGTM
(At least in my quick review)

**Review by screret — APPROVED** [review]

**Comment by IamNotBrianZ** [comment]
What does "1 workflow awaiting approval" means?
Do i miss something?

**Comment by screret** [comment]
> What does "1 workflow awaiting approval" means? Do i miss something?

it means I didn't approve a test build for a thing that doesn't need a test build, nothing to worry about.

**Pull request merged**